### PR TITLE
Move transforms to common

### PR DESCRIPTION
**Pull request by slyalin**

### CODE REVIEW

File: inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp [hunk 1]
```diff
@@ -35,6 +49,30 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
...
+    decomp->add_matcher<ngraph::pass::ConvertReduceSumToPooling>();
+    decomp->add_matcher<ngraph::pass::ConvertMod>();
+    decomp->add_matcher<ngraph::pass::ConvertMinimum>();
+    decomp->add_matcher<ngraph::pass::ConvertSubtract>();
+    decomp->add_matcher<ngraph::pass::ConvertDivide>();
+    decomp->add_matcher<ngraph::pass::ConvertNegative>();
+    decomp->add_matcher<ngraph::pass::ConvertDepthToSpace>();
+    decomp->add_matcher<ngraph::pass::ConvertSpaceToDepth>();
+    decomp->add_matcher<ngraph::pass::BatchNormDecomposition>();
+    decomp->add_matcher<ngraph::pass::PullTransposeThroughFQUp>();
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp:66]: PullTransposeThroughFQUp must be executed after matmul->FC conversion

Comment (slyalin) [reply inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp:69]: There is another call of it after matmul->FC conversion. I put a duplicate here to preserve LPT from FQ->Transpose sequence coming, for example, from MO for some reason. As we don't have (or will prevent) CF for FQ, it makes sense to move as much stuff as possible before FQ to be applicable for CF. It is more theoretical than practical though.

**Review by ilyachur — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
is it still valid?

**Comment by slyalin** [comment]
> is it still valid?

No, surprised that this is not in a draft state/closed.

**Pull request closed without merging**